### ISSUE TITLE
bam file for bap2 input

### ISSUE THREAD

**Issue by yuxia9999** [caleblareau/bap#38]
Hi,
Does the bam file for bap2 input require the reads been duplicates marked, e.g., using Picard to mark the dups but not remove dups? What if I don't run mark dup, the bap2 process will generate some issues, e.g., the jaccard index computations will be too much off?
Thanks a lot!
Yu

**Comment by caleblareau** [comment]
Nope! there will be duplicate mark / removed during the execution of the basic pipeline

**Comment by yuxia9999** [comment]
Got it. Thanks for the quick response!
